### PR TITLE
Simplify params handler additional tests

### PR DESCRIPTION
**Pull request by Jokero**

### CODE REVIEW

File: test/index.js [hunk 1]
```diff
@@ -20,11 +18,37 @@ describe('express-params-handler', function() {
...
+    app.param('date', lib(/^\d{4}-\d{2}-\d{2}$/))
+
+    return app
+  }
+
+  var makeAppWithModifyingHandlerOfParams = function() {
+    var app = makeApp();
+
+    app.param('id', function(req, res, next, value) {
+      req.params.id = value === 'kraken' ? 100 : value;
```
Comment (runk) [inline test/index.js]: Use "current" instead of meaningless "kraken" maybe?

File: test/index.js [hunk 2]
```diff
@@ -67,6 +101,16 @@ describe('express-params-handler', function() {
         })
     })
 
+    it('use actual param value from req.params', function(done) {
+      request(makeAppWithModifyingHandlerOfParams())
+          .get('/by-date/kraken')
```
Comment (runk) [inline test/index.js]: indent one level left

**Comment by runk** [comment]
Nice stuff @Jokero :+1:

**Comment by Jokero** [comment]
Updated

**Comment by runk** [comment]
:rocket:

**Comment by Jokero** [comment]
Pull this branch on master

**Pull request merged**